### ISSUE TITLE
Start with the highest level missions

### ISSUE THREAD

**Issue by AddonWhiner** [TLDRMissions/TLDRMissions#506]
TLDR version: Start by trying to handle the highest level missions so it doesn't waste powerful heroes on the easy ones.

The addon seems to have the ambition to handle every mission using troops instead of heroes. That's alright, but it tends to mean that low level missions that could be handled by pretty much a random composition of C-tier heroes (as a Kyrian, my heroes beat my troops) instead end up using troops and a powerful hero. Then, when we finally reach the high level missions, I cannot beat them as my best heroes have been sent out on easy stuff. So, I end up having 15 C-tier heroes at home who could have instead been doing the easy missions.

It would be better if the addon prioritized mission by highest level (and then by being elite or not).

**Comment by AddonWhiner** [comment]
Never mind, just saw advanced settings.
